pkg/models/dashboards_test.go

```diff
@@ -72,14 +73,14 @@ func TestSlugifyTitle(t *testing.T) {
 	testCases := map[string]string{
 		"Grafana Play Home": "grafana-play-home",
 		"snöräv-över-ån":    "snorav-over-an",
-		"漢字":                "han-zi",      // Hanzi for hanzi
+		"漢字":                "5ryi5a2X",    // "han-zi",      // Hanzi for hanzi
```

Comment:
^^ example where the new slugs will not be great

I have no sense how valuable this is to grafana users

Reply:
I doubt it's particularly valuable, weighed against the associated overhead.  If it did become a problem we could relatively easily expose the ability for the user to specify their own slug via dashboard settings.